stories/wizard/wizard.stories.js

```diff
@@ -0,0 +1,897 @@
+                        <span class="fd-wizard__step-indicator">2</span>
+                        <div class="fd-wizard__label-container">
+                            <span class="fd-wizard__label">
+                                Step 2: Very long label that truncates on the second line
+                            </span>
+                        </div>
+                    </a>
+                    <span class="fd-wizard__connector"></span>
+                </div>
+            </li>
+            <li class="fd-wizard__step fd-wizard__step--upcoming">
+                <div class="fd-wizard__step-wrapper">
+                    <a class="fd-wizard__step-container" tabindex="0" aria-label="Step 3: One line truncates" aria-disabled="true">
+                        <span class="fd-wizard__step-indicator">3</span>
+                        <div class="fd-wizard__label-container fd-wizard__label-container--optional">
+                            <span class="fd-wizard__label">Step 3: One line truncates</span>
+                            <span class="fd-wizard__optional-text">(Optional)</span>
+                        </div>
+                    </a>
+                    <span class="fd-wizard__connector"></span>
+                </div>
+            </li>
+            <li class="fd-wizard__step fd-wizard__step--upcoming">
+                <div class="fd-wizard__step-wrapper">
+                    <a class="fd-wizard__step-container" tabindex="0" aria-label="Step 4: Future Step" aria-disabled="true">
+                        <span class="fd-wizard__step-indicator">4</span>
+                        <div class="fd-wizard__label-container">
+                            <span class="fd-wizard__label">Step 4: Future Step</span>
+                        </div>
+                    </a>
+                </div>
+            </li>
+        </ul>
+    </nav>
+    <section class="fd-wizard__content" id="wizard-section-1" style="min-height: 300px;">
+        <div>
+            Sed fermentum, mi et tristique ullamcorper, sapien sapien faucibus sem, quis pretium nibh lorem malesuada diam. Nulla quis arcu aliquet, feugiat massa semper, volutpat diam. Nam vitae ante posuere, molestie neque sit amet, dapibus velit. Maecenas eleifend tempor lorem. Mauris vitae elementum mi, sed eleifend ligula. Nulla tempor vulputate dolor, nec dignissim quam convallis ut. Praesent vitae commodo felis, ut iaculis felis. Fusce quis eleifend sapien, eget facilisis nibh. Suspendisse est velit, scelerisque ut commodo eget, dignissim quis metus. Cras faucibus consequat gravida. Curabitur vitae quam felis. Phasellus ac leo eleifend, commodo tortor et, varius quam. Aliquam erat volutpat
+        </div>
+        <div class="fd-wizard__next-step">
+            <button class="fd-button fd-button--compact fd-button--emphasized">Step 3</button>
+        </div>
+    </section>
+    <footer>
```

Comment:
Semantically all `<footer>` elements should be enclosed in sectioning content like `<section>`. See https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Sectioning_content